.travis.yml

```diff
@@ -3,23 +3,27 @@ language: generic
 
 sudo: required
 
-script:
-  - ./build.sh
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+jobs:
+  include:
+    - script: ./build.sh
+    - script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+    - stage: GitHub Release
```

Comment:
Then why are you parallising it: `Github Release although happens after build.sh completed not before`